### ISSUE TITLE
[BUG] Single request example not provided

### ISSUE THREAD

**Issue by gcatanese** [OpenAPITools/openapi-generator#15579]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

When a single inline request example is provided by the OpenAPI spec file it is not available during the code generation.

##### openapi-generator version

7.0.0-SNAPSHOT

##### OpenAPI declaration file content or url

See request single example in requestBody
```yaml

      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              required:
                - lat
                - long
              properties:
                lat:
                  type: number
                long:
                  type: number
            example:
              lat: 52.377956
              long: 4.897070
```

##### Steps to reproduce
Define spec with request example (see above) and generate code (any language).

**Comment by ab7022** [comment]
openapi: 3.0.0
info:
  title: Your API
  version: 1.0.0
paths:
  /your-endpoint:
    post:
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              required:
                - lat
                - long
              properties:
                lat:
                  type: number
                long:
                  type: number
            examples:
              example1:
                value:
                  lat: 52.377956
                  long: 4.897070
      responses:
        '200':
          description: OK

**Comment by ab7022** [comment]
`openapi: 3.0.0
info:
  title: Your API
  version: 1.0.0
paths:
  /your-endpoint:
    post:
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              required:
                - lat
                - long
              properties:
                lat:
                  type: number
                long:
                  type: number
            examples:
              example1:
                value:
                  lat: 52.377956
                  long: 4.897070
      responses:
        '200':
          description: OK
`

**Comment by gcatanese** [comment]
Fixed by #15857
